bin/cml-send-comment.test.js

```diff
@@ -30,9 +30,9 @@ describe('CML e2e', () => {
       "Usage: cml-send-comment.js <path> --head-sha <string>
 
       Options:
-        --version   Show version number                                      [boolean]
-        --head-sha  Commit sha
-        -h          Show help                                                [boolean]"
+        --version     Show version number                                    [boolean]
+        --commit-sha  Commit sha
```

Comment:
This looks like an API change. It is better to keep the old option for backward compatibility with some warning messages and depreciate later (in a few months).

Reply:
sounds good